### PR TITLE
Cockpit 182

### PR DESCRIPTION
**Pull request by Gundersanne**
https://gundersanne.github.io/cockpit-project.github.io/blog/cockpit-182.html

### CODE REVIEW

File: _posts/2018-11-14-cockpit-182.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### PackageKit page improvements
+
+Update severity is now indicated by a tooltip. The severity shown is equal to
+the severity of the update with the highest severity among the available updates
+for a package.
+
+![PackageKit severity tooltip](/images/packagekit-severity-tooltips.png)
+
+Furthermore if the system is not registered the PackageKit page will now display
+this clearly.
```
Comment (martinpitt) [inline _posts/2018-11-14-cockpit-182.md]: The page is called "Software Updates", not "PackageKit".

To clarify, this empty state has always existed, but only if there were really no pending updates (e. g. from repos that don't require subscription). How about:

If the system is not registered,  the Software Updates page now always points this out. This avoids confusion if there are some available updates from third-party repositories, but the system as a whole still does not receive updates.

Comment (Gundersanne) [reply _posts/2018-11-14-cockpit-182.md]: Ah, then I misunderstood that. I'll change it thanks!

**Comment by martinpitt** [comment]
Asking @garrett for a review too, as our resident native speaker and "friendly lingo" expert :smile:

File: _posts/2018-11-14-cockpit-182.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 182.
+
+### Machines: Add libvirt connection choice in VM creation dialog
+
+The VM creation dialog now offers different connections to the libvirt
+daemon. The options result in either a user session URI or a system URI.
```
Comment (garrett) [inline _posts/2018-11-14-cockpit-182.md]: Instead of "the libvirt daemon", consider either:

- "the libvirt service"
- "libvirt"

Comment (garrett) [reply _posts/2018-11-14-cockpit-182.md]: Also, remove the duplicate "URI" and "a":

```suggestion
daemon. The options result in either a user session or system URI.
```

File: _posts/2018-11-14-cockpit-182.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+![Creation dialog connection
+choice](/images/machines-libvirt-create-connection-choice.png)
+
+_Thanks to Katerina Koukiou for this improvement!_
+
+### Software Updates page improvements
+
+Update severity is now indicated by a tooltip. The severity shown is equal to
```
Comment (garrett) [inline _posts/2018-11-14-cockpit-182.md]:
```suggestion
Update severity icons now have tooltips. The severity shown is equal to
```

File: _posts/2018-11-14-cockpit-182.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+the severity of the update with the highest severity among the available updates
+for a package.
+
+![Software Updates severity tooltip](/images/packagekit-severity-tooltips.png)
+
+If the system is not registered, the Software Updates page now always points
+this out. This avoids confusion if there are some available updates from
```
Comment (garrett) [inline _posts/2018-11-14-cockpit-182.md]:
```suggestion
this out. This avoids confusion if there are available updates from
```

**Comment by garrett** [comment]
I made suggestions where possible. However, it's not possible to make suggestions that span lines.

I suggest using your editor's soft wrap instead of hard wrapping Markdown for easier merging and changes in the future. :grin:

**Pull request merged**